src/elements/core/mixins/panel-mixin.ts

```diff
@@ -24,33 +21,6 @@ export const SbbPanelMixin = <T extends AbstractConstructor<LitElement>>(
 
     /** Whether the unselected panel has a border. */
     @property({ reflect: true, type: Boolean }) public borderless = false;
-
```

Comment:
Is this mixin still needed? Given it's providing just two properties it's starting to look a bit like over-engineering. What do you think? 🤔

Reply:
I think yes, also because we have the corresponding style file. Could you live with keeping it? ;)

Reply:
Yep, it makes sense on a structural level